### PR TITLE
stolostron/backlog#24626 rename infra env

### PR DESCRIPTION
**Pull request by cadawson**

### CODE REVIEW

File: multicluster_engine/cluster_lifecycle/add_hosts_host_inventory.adoc [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
-You can use the {product-title} console to add hosts to an infrastructure environment. Adding the hosts makes it easier to select the already-configured hosts when you are creating a cluster. 
+You can use the console to add hosts to a host inventory. Adding the hosts makes it easier to select the already-configured hosts when you are creating a cluster. 
 
 Complete the following steps to add a host:
 
-. From the navigation, select *Infrastructure* > *Infrastructure environments*.
+. From the navigation, select *Infrastructure* > *Host inventory*.
 
-. Select the infrastructure environment where you want to add the host to view its settings.
+. Select the host inventory where you want to add the host to view its settings.
```
Comment (jelkosz) [inline multicluster_engine/cluster_lifecycle/add_hosts_host_inventory.adoc:10]: this actually needs to stay "infrastructure environment". The entities are still called "infrastructure environment", just the left menu item changed to "host inventory".

File: multicluster_engine/cluster_lifecycle/add_hosts_host_inventory.adoc [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
 
-. Select the _Hosts_ tab to view the hosts that are already added to that environment and to add a host. Available hosts might take a few minutes to appear in the table. 
+. Select the _Hosts_ tab to view the hosts that are already added to that host inventory and to add a host. Available hosts might take a few minutes to appear in the table.
```
Comment (jelkosz) [inline multicluster_engine/cluster_lifecycle/add_hosts_host_inventory.adoc:12]: mmm, technically it is "environment" but it does make sense to have "host inventory" too here, so I dont have a strong feeling about this part.

File: multicluster_engine/cluster_lifecycle/add_hosts_host_inventory.adoc [hunk 3]
```diff
@@ -41,4 +41,4 @@ When you add a host by using the BMC option, the host is automatically approved.
 
 .. Select *Upload* and navigate to the completed YAML template to add multiple hosts. 
 
-You can now create an on-premises cluster on this infrastructure environment. See xref:../multicluster_engine/create_cluster_on_prem.adoc#creating-a-cluster-on-premises[Creating a cluster in an on-premises environment] for more information about creating a cluster.
+You can now create an on-premises cluster on this host inventory. See xref:../cluster_lifecycle/create_cluster_on_prem.adoc#creating-a-cluster-on-premises[Creating a cluster in an on-premises environment] for more information about creating a cluster.
```
Comment (jelkosz) [inline multicluster_engine/cluster_lifecycle/add_hosts_host_inventory.adoc:44]: I would keep infra env also here.

File: multicluster_engine/cluster_lifecycle/create_cluster_on_prem.adoc [hunk 4]
```diff
@@ -16,7 +16,7 @@ You can also test the procedure of provisioning multiple single-node OpenShift c
 See the following prerequisites before creating a cluster in an on-premises environment:
 
 * You must have a deployed {product-title-short} hub cluster on {ocp-short} version 4.9, or later.
-* You need a configured infrastructure environment with configured hosts. 
+* You need a configured host inventory with configured hosts.
```
Comment (jelkosz) [inline multicluster_engine/cluster_lifecycle/create_cluster_on_prem.adoc:19]: technically you need an infra env under host inventory. Not sure how to phrase it here.

File: multicluster_engine/cluster_lifecycle/create_host_inventory.adoc [hunk 5]
```diff
@@ -1,28 +1,28 @@
-[#creating-an-infrastructure-environment]
-= Creating an infrastructure environment
+[#creating-a-host-inventory]
+= Creating a host inventory
 
-You can use the {product-title} console to create an infrastructure environment to manage your hosts and create clusters on those hosts.
+You can use the console to create a host inventory to manage your hosts and create clusters on those hosts.
```
Comment (jelkosz) [inline multicluster_engine/cluster_lifecycle/create_host_inventory.adoc:4]: here for sure infra env

File: multicluster_engine/cluster_lifecycle/create_host_inventory.adoc [hunk 6]
```diff
@@ -1,28 +1,28 @@
...
-* <<infra-env-prerequisites,Prerequisites>>
+* <<host-inv-prerequisites,Prerequisites>>
 * <<enable-cim,Enable Central Infrastructure Management service>>
-* <<creating-your-infra-env-with-the-console,Creating your infrastructure environment with the console>>
-* <<accessing-your-infra-env,Accessing your infrastructure environment>>
+* <<creating-your-host-inv-with-the-console,Creating your host inventory with the console>>
+* <<accessing-your-host-inv,Accessing your host inventory>>
 
-Infrastructure environments support the following features:
+Host inventories support the following features:
```
Comment (jelkosz) [inline multicluster_engine/cluster_lifecycle/create_host_inventory.adoc:11]: same, infra env here.

File: multicluster_engine/cluster_lifecycle/create_host_inventory.adoc [hunk 7]
```diff
@@ -1,28 +1,28 @@
...
 - Dual stack: Deploy clusters that have both IPv4 and IPv6 addresses. Dual stack uses the `OVN-Kubernetes` networking implementation to support multiple subnets. 
 - Add remote worker nodes: Add remote worker nodes to your clusters after they are created and running, which provides flexibility of adding nodes in other locations for backup purposes.
 - Static IP using NMState: Use the NMState API to define static IP addresses for your environment.
 
-[#infra-env-prerequisites]
+[#host-inv-prerequisites]
 == Prerequisites
 
-See the following prerequisites before creating an infrastructure environment:
+See the following prerequisites before creating a host inventory:
```
Comment (jelkosz) [inline multicluster_engine/cluster_lifecycle/create_host_inventory.adoc:22]: same, infra env here.

File: multicluster_engine/cluster_lifecycle/create_host_inventory.adoc [hunk 8]
```diff
@@ -107,18 +107,18 @@ agentserviceconfig.agent-install.openshift.io/agent created
...
-== Creating your infrastructure environment with the console
+[#creating-your-host-inv-with-the-console]
+== Creating your host inventory with the console
 
-To create an infrastructure environment from the {product-title-short} console, complete the following steps:
+To create a host inventory from the console, complete the following steps:
 
-. From the navigation menu, navigate to *Infrastructure* > *Infrastructure environments* and click *Create infrastructure environment*.
-. Add the following information to your infrastructure environment settings: 
+. From the navigation menu, navigate to *Infrastructure* > *Host inventory* and click *Create host inventory*.
```
Comment (jelkosz) [inline multicluster_engine/cluster_lifecycle/create_host_inventory.adoc:115]: ... Infrastructure > Host inventory  and click Create infrastructure environment

File: multicluster_engine/cluster_lifecycle/create_host_inventory.adoc [hunk 9]
```diff
@@ -107,18 +107,18 @@ agentserviceconfig.agent-install.openshift.io/agent created
...
+. Add the following information to your host inventory settings:
```
Comment (jelkosz) [inline multicluster_engine/cluster_lifecycle/create_host_inventory.adoc:116]: infra env

File: multicluster_engine/cluster_lifecycle/create_host_inventory.adoc [hunk 10]
```diff
@@ -107,18 +107,18 @@ agentserviceconfig.agent-install.openshift.io/agent created
...
+To create a host inventory from the console, complete the following steps:
 
-. From the navigation menu, navigate to *Infrastructure* > *Infrastructure environments* and click *Create infrastructure environment*.
-. Add the following information to your infrastructure environment settings: 
+. From the navigation menu, navigate to *Infrastructure* > *Host inventory* and click *Create host inventory*.
+. Add the following information to your host inventory settings: 
 +
-* Name: A unique name for your environment. 
-* Network type: Specifies which types of hosts can be added to your environment. You can only use the static IP option when you are using bare metal hosts. 
+* Name: A unique name for your host inventory.
```
Comment (jelkosz) [inline multicluster_engine/cluster_lifecycle/create_host_inventory.adoc:118]: infra env

File: multicluster_engine/cluster_lifecycle/create_host_inventory.adoc [hunk 11]
```diff
@@ -107,18 +107,18 @@ agentserviceconfig.agent-install.openshift.io/agent created
...
 
-. From the navigation menu, navigate to *Infrastructure* > *Infrastructure environments* and click *Create infrastructure environment*.
-. Add the following information to your infrastructure environment settings: 
+. From the navigation menu, navigate to *Infrastructure* > *Host inventory* and click *Create host inventory*.
+. Add the following information to your host inventory settings: 
 +
-* Name: A unique name for your environment. 
-* Network type: Specifies which types of hosts can be added to your environment. You can only use the static IP option when you are using bare metal hosts. 
+* Name: A unique name for your host inventory. 
+* Network type: Specifies which types of hosts can be added to your host inventory. You can only use the static IP option when you are using bare metal hosts.
```
Comment (jelkosz) [inline multicluster_engine/cluster_lifecycle/create_host_inventory.adoc:119]: infra env

File: multicluster_engine/cluster_lifecycle/create_host_inventory.adoc [hunk 12]
```diff
@@ -107,18 +107,18 @@ agentserviceconfig.agent-install.openshift.io/agent created
...
-* Network type: Specifies which types of hosts can be added to your environment. You can only use the static IP option when you are using bare metal hosts. 
+* Name: A unique name for your host inventory. 
+* Network type: Specifies which types of hosts can be added to your host inventory. You can only use the static IP option when you are using bare metal hosts. 
 * Location: Specifies the geographic location of the host. The geographic location can be used to easily determine where your data on a cluster is stored when you are creating the cluster. 
-* Labels: Optional field where you can add labels to the infrastructure environment so you can more easily find and group the environment with other environments that share a characteristic. The selections that you made for the network type and location are automatically added to the list of labels.
+* Labels: Optional field where you can add labels to the host inventory so you can more easily find and group the host inventory with other host inventories that share a characteristic. The selections that you made for the network type and location are automatically added to the list of labels.
```
Comment (jelkosz) [inline multicluster_engine/cluster_lifecycle/create_host_inventory.adoc:121]: infra env everywhere here

File: multicluster_engine/cluster_lifecycle/create_host_inventory.adoc [hunk 13]
```diff
@@ -129,10 +129,10 @@ To create an infrastructure environment from the {product-title-short} console,
 
 ** No Proxy domains: A comma-separated list of domains that should bypass the proxy. Begin a domain name with a period `.` to include all of the subdomains that are in that domain. Add and asterisk `*` to bypass the proxy for all destinations. 
 
-You can now continue by adding hosts to your infrastructure environment. 
+You can now continue by adding hosts to your host inventory.
```
Comment (jelkosz) [inline multicluster_engine/cluster_lifecycle/create_host_inventory.adoc:132]: infra env

**Review by jelkosz — CHANGES_REQUESTED** [review]
There seems to be a misunderstanding about the extent of the change in console. The entity infrastructure environment has not been renamed to host inventory. Only the menu item in the left menu.

E.g. the host inventory contains infrastructure environments:
image

**Comment by cadawson** [comment]
Starting over

**Pull request closed without merging**